libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/ComposeIT.kt

```diff
@@ -755,6 +755,66 @@ class ComposeIT : KGPBaseTest() {
         }
     }
 
+    @DisplayName("CMP-9167 verification")
+    @GradleAndroidTest
+    @AndroidTestVersions(minVersion = TestVersions.AGP.AGP_811, maxVersion = TestVersions.AGP.AGP_811)
+    @OtherGradlePluginTests
+    @TestMetadata("composeMultiplatform")
+    fun testComposeDefaultValueParamStubs(
+        gradleVersion: GradleVersion,
+        agpVersion: String,
+        providedJdk: JdkVersions.ProvidedJdk
+    ) {
+        project(
+            projectName = "composeMultiplatform",
```

Comment:
Could you use build script injection approach in this test? See `BuildScriptInjectionIT` for examples.